File: gpio/led.c
//blinking an Led
#include <wiringPi.h>
#include <stdio.h>

#define LedPin 0  //aka pin 11 On BOARD 

int main(void){
	wiringPiSetup(); // initialize wiring
	if (wiringPiSetup()==-1){  //when initialize wiring failed, print message to screen(unnecessary)
		printf("setup writingPi failed!");
		return 1;
	}

	pinMode(LedPin, OUTPUT);  // LedPin will send signal to the led
	while(1){
		digitalWrite(LedPin, LOW);//send low signal to led through pin11
		printf("led on...\n");
		delay(500);
		digitalWrite(LedPin,HIGH);//send high signal to led through pin11
		printf("led off...\n");
		delay(500);
	}
	return 0;
}

	

File: gpio/led.py
# blinking an Led
import RPi.GPIO as GPIO
import time

while True:
	GPIO.setmode(GPIO.BOARD)  # use the pin number on board
	GPIO.setup(11,GPIO.OUT) # pin11 will send signal to led 
	GPIO.output(11,GPIO.HIGH)  # send high signal to led through pin11 
	print("led off...")
	time.sleep(0.5)            
	GPIO.output(11,GPIO.LOW)    #send low signal to led through pin11
	print("led on...")
	time.sleep(0.5)
	GPIO.cleanup()

